Tools/Plotfile/fextract.cpp

```diff
@@ -155,8 +148,21 @@ void main_main()
         kloc = (hi0.z-lo0.z+1)/2 + lo0.z;
     }
 
+    if (xcoord > -1.e-36 && AMREX_SPACEDIM >= 1) {
```

Comment:
What's special about `-1.e-36`?  Was it meant to be `-1.e36` or `std::numeric_limit<Real>::lowest()`?

Reply:
I dunno, I just copied what was there.  Do a git blame and ask the offender :)

Reply:
I asked myself and concluded that it was a typo.  :facepalm: Could we change all these to `-1.e36`?